routes/index.js

```diff
@@ -108,12 +112,15 @@ router.post('/login', csrfProtection, loginValidators,
           // If the password hashes match, then login the user name!
           // and redirect them to the home route.
           loginUser(req, res, user, next);
-          return;
+        }else{
+          // Otherwise display an error message to the user.
+          errors.push('Login failed for the provided user name and password, quest-taker');
+          // return;
         }
       }
 
       // Otherwise display an error message to the user.
-      errors.push('Login failed for the provided user name and password, quest taker');
+      // errors.push('Login failed for the provided user name and password, quest taker');
```

Comment:
Please remove this commented out code and comment.

Reply:
Removed!